node/src/components/linear_chain_fast_sync.rs

```diff
@@ -602,3 +616,21 @@ where
             move || Event::GetBlockHeightResult(block_height, BlockByHeightResult::Absent(cloned)),
         )
 }
+
+/// Reads the `LinearChainSync's` state from storage, if any.
+/// Panics on deserialization errors.
+pub(crate) fn read_init_state(
+    _storage: &Storage,
+    _chainspec: &Chainspec,
+) -> Result<Option<State>, storage::Error> {
+    panic!("not yet implemented for fast sync")
+}
+
+/// Cleans the linear chain state storage.
+/// May fail with storage error.
+pub(crate) fn clean_linear_chain_state(
+    _storage: &Storage,
+    _chainspec: &Chainspec,
+) -> Result<(), storage::Error> {
+    panic!("not yet implemented for fast sync")
```

Comment:
Let's just have the callsite for this be behind `#[cfg(not(feature = "fast-sync"))]` so we can get rid of this stub

Reply:
Done in [`087dddd` (#904)](https://github.com/CasperLabs/casper-node/pull/904/commits/087dddd78b6b23012eca2d7f765adf874623c248).